paypalscript.py

```diff
@@ -1,16 +1,49 @@
 #!/usr/bin/env python3
+import argparse
 import csv
+import os.path
+
+#
+# Parse and set Input and Output Files
+#
+
+parser = argparse.ArgumentParser(description='Takes a PayPal CSV export file and generates a GnuCash formatted CSV import file.  Default input file name is Paypal Export.csv.  Defaults output file name is output.csv')
+parser.add_argument('-i','--input', help='Input file name',required=False)
+parser.add_argument('-o','--output',help='Output file name', required=False)
+args = parser.parse_args()
+
+inputFile = args.input
+if not bool(inputFile):
+    inputFile = 'Paypal Export.csv'
+
+if not os.path.exists(inputFile.strip('\n')):
+    print("Input file does not exist: " + inputFile)
+    quit()
+
+print("Input File:  " + inputFile)    
+
+outputFile = args.output
+if not bool(outputFile):    
+    outputFile = "Output.csv"
+
+print("Output File: " + outputFile)
+
+#
+# Parse input file and create output file
+#
 
 mapping = {"Date": "Date", "Name": "Name", "Amount": "Net", "Description": "Type"}
 
 def strip_row(row):
     return {k: row[v] for k,v in mapping.items()}
 
-with open("Paypal Export.csv", 'r', encoding="utf-8-sig") as filein:
+with open(inputFile, 'r', encoding="utf-8-sig") as filein:
     csv_filein = [strip_row(row) for row in csv.DictReader(filein) if row["Balance Impact"] == "Credit"]
```

Comment:
Minor code smell: Hardcoded strings can cause issues if the data structure changes. Consider using a variable with documentation for what it is, what it does, and why they may need to change it

Reply:
While a valid critique of the code, it doesn't belong on this change set and should be addressed elsewhere. This also applies to the next 3 comments.

Reply:
Opened Issue #6 to resolve